### PR TITLE
Fcl 272/further UI changes

### PR DESCRIPTION
**Pull request by jlhdxw**
## Changes in this PR:

The last UI changes for the EUI. Includes:

 - [fix judgment date over 2 lines](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/4ec319d9e81dd05746b827dea4bed9d6b1a85ee3)
 - [make hold confirmation text clearer](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/d188fe92db3e54a677de4c73562a8a1bf7057a18)
 - [improve on hold confirmation page](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/5e7a252557e5aee7108a0299d0cd7e540b122e1b)
 - [simplify the email submitter cta on hold confirmation](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/ffade566f7c475a3a952137295352e9279e8b8a3)
 - [improve unhold flow UI](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/8ae6ff8fc7be9a81ee4dda93881be97ef4238e8e)
 - [improve publish flow UI](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/e75daf1a5bc1c976f231b852be4a0ba295659ea2)
 - [improve published success page](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/56190974b11322566c5623cb1cfe5cda95769867)
 - [change unpublished link to go to published docs](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/ef4663e1e3060c0c96660951a43adfd0c6e3d898)
 - [improve history list](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/a03a0c92d73d568e7f8e68a74781ce22b2a0702a)
 - [improve the version viewing text on document view](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/46335666fbb51298b9048005dba85d57c8221d77)
 - [move status up from sidebar](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/e8668eacb438c1b5cfad1d58c72f3a721746368e)
 - [update corrected NCN notification](https://github.com/nationalarchives/ds-caselaw-editor-ui/commit/45814965c0aeab5b1c643b4b3f33522f419bcc5c)

## Jira card / Rollbar error (etc)

https://national-archives.atlassian.net/browse/FCL-272

## Screenshots of UI changes:

image

image

image

image

image

image

image

image

image

image

### CODE REVIEW

File: ds_caselaw_editor_ui/sass/includes/_judgment_metadata_panel.scss [hunk 1]
```diff
@@ -11,17 +11,19 @@
...
+    flex-wrap: wrap;
+    column-gap: $space-8;
+    row-gap: $space-2;
 
     li {
-      float: left;
       width: 100%;
       @media (min-width: $grid-breakpoint-medium) {
-        width: 50%;
+        width: 48%;
```
Comment (dragon-dxw) [inline ds_caselaw_editor_ui/sass/includes/_judgment_metadata_panel.scss:22]: Feels a little magical.

File: ds_caselaw_editor_ui/templates/judgment/unhold.html [hunk 2]
```diff
@@ -5,7 +5,9 @@
 {% endblock judgment_header %}
 {% block judgment_content %}
   {% load i18n %}
-  <h2>Take {{ judgment.name }} off hold?</h2>
+  <h3 class="judgment-component__confirmation-title">
```
Comment (dragon-dxw) [inline ds_caselaw_editor_ui/templates/judgment/unhold.html]: I am questioning how much we actually want an intersitial for things that are currently on hold. But not a problem for this PR.

File: ds_caselaw_editor_ui/templates/judgment/history.html [hunk 3]
```diff
@@ -51,7 +51,13 @@ <h2>Document version history</h2>
           {% for version in document.versions_as_documents %}
             <tr>
               <td>
-                <a href="{% url 'full-text-html' document_uri %}?version_uri={{ version.uri }}">Version {{ version.version_number }}</a>
+                <a href="{% url 'full-text-html' document_uri %}?version_uri={{ version.uri }}">
+                  {% if forloop.first %}
+                    <strong>Latest version</strong>
```
Comment (dragon-dxw) [inline ds_caselaw_editor_ui/templates/judgment/history.html]: I think I'd like a version number here, to be explicit. Maybe "Version 6 (Latest)"?

File: judgments/utils/__init__.py [hunk 4]
```diff
@@ -184,3 +184,12 @@ def _build_related_document_uri(document: Document) -> str:
     if isinstance(document, PressSummary):
         return document.uri.removesuffix(press_summary_suffix)
     return document.uri + press_summary_suffix
+
+
+def get_corrected_ncn_url(document) -> str | None:
```
Comment (dragon-dxw) [inline judgments/utils/__init__.py:189]: Pondering if I should write `Optional[str]` this way too.

**Review by dragon-dxw — APPROVED** [review]

File: ds_caselaw_editor_ui/templates/judgment/history.html (hunk 3 above)
Comment (jlhdxw) [reply ds_caselaw_editor_ui/templates/judgment/history.html]: Agreed - will update

File: ds_caselaw_editor_ui/sass/includes/_judgment_metadata_panel.scss (hunk 1 above)
Comment (jlhdxw) [reply ds_caselaw_editor_ui/sass/includes/_judgment_metadata_panel.scss:22]: Ideally I'd like to restyle this entirely as the styling here is a bit magical 😄 I think i'll save that for another clean up PR.

**Pull request merged**